### DISCUSSION TITLE
verification of arguments of type enum

### DISCUSSION THREAD

**Discussion by duduarbel** [capnproto/capnproto#2200] [Q&A]
How can I add a verification that a Python argument for a function is actually of an capnp enum type?
I could not find a way to verify this. For example - I could not differentiate one enum from another, as `col` is actually an integer
```
enum Color {
  RED;
  GREEN;
  BLUE;
}
def foo(col: Color):
     # how to verify that col is actually of type Color?
```

**Comment by kentonv** [comment]
It sounds like this is a question about pycapnp? It might make sense to post it directly on the pycapnp repo. (I am personally not that familiar with pycapnp, sorry.)
